src/main/scala/fi/oph/koski/opiskeluoikeus/OpiskeluoikeusQueries.scala

```diff
@@ -16,7 +16,7 @@ import javax.servlet.http.HttpServletRequest
 import org.scalatra._
 import rx.lang.scala.Observable
 
-trait OpiskeluoikeusQueries extends KoskiSpecificApiServlet with Logging with GlobalExecutionContext with ObservableSupport with ContentEncodingSupport with Pagination with HasKoskiSpecificSession {
+trait OpiskeluoikeusQueries extends KoskiSpecificApiServlet with Logging with ObservableSupport with ContentEncodingSupport with Pagination with HasKoskiSpecificSession {
```

Comment:
Jos oikein päättelen, niin implisiittiset execution contextit vaikuttavat siihen, missä poolissa ajetaan kyseisen luokan käynnistämät rinnakkaiset operaatiot.

Tuleeko näiden GlobalExecutionContext-perintöjen poiston vuoksi jotain muutoksia siihen, missä koodia ajetaan, vai ovatko vain refaktorointeja ja perintä tulee kantaluokan kautta, tai implisiittistä arvoa ei käytetä lainkaan kyseisessä luokassa tai sen kutsumassa koodissa?

Reply:
Tästä olikin äsken puhetta.

Kävin siis ennen muutosten tekoa läpi `GlobalExecutionContext`in sisällyttävät luokat katsoen IDEAn avulla käytetäänkö traitin implicit poolia oikeasti. On tietenkin mahdollista, että IDEA on näyttänyt jossain kohti väärin, joten tsekkaan näistä vielä kerran perinnän kautta mahdollisesti tulevat poolit. Löytyikin jo yksi paikka jossa `GlobalExecutionContext` on käytössä kantakyselyjen ajamiseen. Tähän kohtaan en siis ollut aiemmin koskenutkaan, eli nämä on olleet sekaisin jo valmiiksi.